pytrip/vdx.py

```diff
@@ -556,8 +556,8 @@ def create_cylinder(cube, name, center, radius, depth):
     p = list(zip(center[0] + radius * np.cos(t), center[1] + radius * np.sin(t)))
 
     for i in range(0, cube.dimz):
-        z = i * cube.slice_distance
-        if center[2] - depth / 2 <= z <= center[2] + depth / 2:
+        z = i * cube.slice_distance + cube.zoffset
```

Comment:
Make proper documentation how the VDX data is being handled, where offsets are added/subtracted and what is saved/read to/from the VDX files.
Investigate also so called z_table.